website/content/docs/upgrading/plugins.mdx

```diff
@@ -50,39 +52,33 @@ an auth plugin, just replace all usages of `secrets` or `secret` with `auth`.
         -version=v1.0.1 \
         secret \
         my-secret-plugin
+    Success! Registered plugin: my-secret-plugin
     ```
 
-1. Tune the existing mount to configure it to use the newly registered version.
+1. Set the new version as the cluster's pinned version.
 
    ```shell-session
-   $ vault secrets tune -plugin-version=v1.0.1 my-secret-plugin
+   $ vault write sys/plugins/pins/secret/my-secret-plugin version=v1.0.1
    ```
 
-1. If you wish, you can check the updated configuration. Notice the "Version" is
-   now different from the "Running Version".
+1. Finally, trigger a global [plugin reload](/vault/docs/commands/plugin/reload)
+   to reload all instances of the plugin.
 
-   ```shell-session
-   $ vault secrets list -detailed
-   ```
+    ```shell-session
+    $ vault plugin reload -type=secret -plugin=my-secret-plugin -scope=global
+    Success! Reloading plugin: my-secret-plugin, reload_id: 98b1e875-4217-745d-07f2-93d14219fb3c
+    ```
 
-1. Finally, trigger a [plugin reload](/vault/docs/commands/plugin/reload) to reload all
-   mounted backends using that plugin or a subset of the mounts using that plugin
-   with either the `plugin` or `mounts` flag respectively.
+1. **Optional:** Check the "Running Version" field to verify the new version is
+    running:
 
     ```shell-session
-    $ vault plugin reload -plugin my-secret-plugin
+    $ vault secrets list -detailed
     ```
 
 Until the last step, the mount will still run the first version of `my-secret-plugin`. When
```

Comment:
Maybe `Until the reload step,` instead? Since reload is no longer the last step listed here.

Reply:
Thanks, good catch